lively.ide/studio/controls/layout.cp.js

```diff
@@ -486,13 +487,15 @@ const LayoutControl = component(PropertySection, {
       part(AddButton, {
         master: { auto: AddButton, hover: PropertyLabelHovered },
         name: 'vertical',
+        tooltip: 'Position Submorphs Vertically',
```

Comment:
I remember that a long time ago there was a discussion within the lively.next team about not using the term "morph" in user facing controls. Basically the idea was that a Designer or someone who does not understand code also does not understand what a morph is.

Reply:
> I remember that a long time ago there was a discussion within the lively.next team about not using the term "morph" in user facing controls. Basically the idea was that a Designer or someone who does not understand code also does not understand what a morph is.

That makes sense! Changed it! :rocket: